### ISSUE TITLE
Optimize when to show the `Syncing Your Entries` spinner

### ISSUE THREAD

**Issue by sashei** [mozilla-lockbox/lockbox-ios#755]
### Acceptance criteria

Tune the Sync spinner so that it doesn't flash in & out so frequently.

**Comment by changecourse** [comment]
Agreed – my hope would be that this would keep the syncing indicator from appearing each time someone navigates to the list or pops between our app and another app?

**Comment by sashei** [comment]
Ooooooops I meant to file this in Android... it's already happening >.<

@changecourse did you have something in particular in mind on iOS related to the spinner? Happy to re-purpose this issue.

**Comment by changecourse** [comment]
Oh, fair enough, heh. 

My observation on iOS is that I've been seeing the indicator a lot lately, between app state switching, or even when jumping from settings back to the entry list... 

I'm wondering if we could repurpose this issue to consider a max number of sync checks during a given amount of time, to prevent what I imagine is a bunch of empty, "no, nothing has changed" moments with the indicator flashes. Perhaps an initial check, then no checks for approx 5min, unless triggered by a manual pull to refresh, or via a complete closing/dismissing and reopening of the app?
